pmoves/tests/smoke/test_supabase_selfhosted.py

```
@@ -33,32 +33,66 @@
+            timeout=5,
+        )
+        return result.returncode == 0
+    except (FileNotFoundError, subprocess.TimeoutExpired):
+        return False
+
+
+def _container_running(name: str) -> bool:
+    """Check if a Docker container is running."""
+    try:
+        result = subprocess.run(
+            ["docker", "ps", "--filter", f"name={name}", "--format", "{{{{.Status}}}}"],
```

Ruff (0.15.5)

[error] Starting a process with a partial executable path

(S607)